shouldForwardProp/should-forward-prop.d.ts

```diff
@@ -0,0 +1,9 @@
+export = gooberGlobal;
```

Comment:
Nit: I think this should be changed to 
```suggestion
export = gooberShouldForwardProp;
```

right?